packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -128,7 +127,7 @@ test.serial(
       bridgeUtils.setBech32Prefix('noble');
 
       const materials = buildProposal(
-        '@agoric/builders/scripts/fast-usdc/start-fast-usdc.build.js',
+        'fast-usdc-deploy/src/start-fast-usdc.build.js',
```

Comment:
note to self: how does this work? Did the `buildProposals` API change from taking module specifiers to something else?

Reply:
this is a bug but it isn't breaking anything because it's wrapped in,
```
    const freshDeploy = false; // prop 87 done
    if (freshDeploy) {
```